lib/chef/chef_fs/file_system/repository/chef_repository_file_system_root_dir.rb

```diff
@@ -163,11 +163,11 @@ def fs_description
           def root_dir
             existing_paths = root_paths.select { |path| File.exists?(path) }
             if existing_paths.size > 0
-              MultiplexedDir.new(existing_paths.map do |path|
+              MultiplexedDir.new(existing_paths.map { |path|
                 dir = FileSystemEntry.new(name, parent, path)
                 dir.write_pretty_json = !!write_pretty_json
                 dir
-              end)
+              })
```

Comment:
This looks correct but i had to go through 4 levels of super calls in the FileSystemEntry.new initializer to determine that none of them had side effects.  And while i try to put initializers on a diet whenever i come across fat ones with side-effects that definitely can't be trusted in our codebase.  (That many levels of inheritance is another thing, but we're well into crazy sections of the chef codebase here).